### PR TITLE
Old dataset landing pages now can reference new filterable datasets

### PR DESCRIPTION
**Pull request by dandwal**
### What

Old dataset landing pages now can reference new filterable datasets
#### Dependencies
Requires this PR branch in zebedee: https://github.com/ONSdigital/zebedee/pull/280

### How to review

Checkout branch, link a filterable dataset, this should work (note without the
### Who can review

Anyone except me

### CODE REVIEW

File: src/legacy/js/functions/_externalLinkAccordionSection.js [hunk 1]
```diff
@@ -7,6 +7,7 @@
  */
 
 function renderExternalLinkAccordionSection(collectionId, data, field, idField) {
+    console.log(collectionId, data, field, idField);
```
[the remark this answers is not in the document]
Comment (jondewijones) [reply src/legacy/js/functions/_externalLinkAccordionSection.js]: Fixed

File: src/legacy/js/functions/_externalLinkAccordionSection.js [hunk 2]
```diff
@@ -130,5 +139,43 @@ function renderExternalLinkAccordionSection(collectionId, data, field, idField)
...
+        $('#uri-input').change(function () {
+            uri = $('#uri-input').val();
+        });
+
+        $('.btn-uri-get').off().click(function () {
+            try {
+                var parsedURL = new URL(uri)
+            }
+            catch(error) {
+                sweetAlert("That doesn't look like a valid URL. Plese include all parts of the URL e.g. 'http://', 'www', etc.");
```
[the remark this answers is not in the document]
Comment (jondewijones) [reply src/legacy/js/functions/_externalLinkAccordionSection.js]: Fixed both

File: src/legacy/js/functions/_externalLinkAccordionSection.js [hunk 3]
```diff
@@ -96,7 +105,7 @@ function renderExternalLinkAccordionSection(collectionId, data, field, idField)
             data[field] = [];
         }
 
-        var linkData = {title: title, uri: uri};
+        var linkData = {title: title, uri: uri, showTitleField: true};
```
[the remark this answers is not in the document]
Comment (jondewijones) [reply src/legacy/js/functions/_externalLinkAccordionSection.js]: ¯\_(ツ)_/¯

File: src/legacy/js/functions/_relatedItemAccordionSection.js [hunk 4]
```diff
@@ -156,7 +158,7 @@ function initialiseRelatedItemAccordionSection(collectionId, data, templateData,
         var viewModel = {hasLatest: false}; //Set to true if 'latest' checkbox should show
         var latestCheck; //Populated with true/false later to check state of checkbox
 
-        if (idField === 'article' || idField === 'bulletin' || idField === 'articles' || idField === 'bulletins' || idField === 'document' || idField === 'highlights') {
+        if (idField === 'article' || idField === 'bulletin' || idField === 'articles' || idField === 'bulletins' || idField === 'document' || idField === 'highlights' || idField === 'filterable-datasets') {
```
[the remark this answers is not in the document]
Comment (jondewijones) [reply src/legacy/js/functions/_relatedItemAccordionSection.js]: Don't disagree with the points but it is consistent with others if we do it this way. I don't think it's that valuable to spend the time refactoring all the instances of it in parts of the codebase that we want to get rid of anyways. And it's not my change ;)

**Comment by dandwal** [comment]
I approve of these changes - but the owner of initial PR can't approve due to protections

**Pull request merged**